demo/screens/rating.py

```diff
@@ -1,7 +1,7 @@
 from kivy.lang.builder import Builder
 from kivy.uix.screenmanager import Screen
 
-from akivymd.uix.rating import AKRating
+from kivymd.components.akivymd.uix.rating import AKRating
```

Comment:
t is not right. Because my pull request assumes that the demo package is using the root package `akivimd` and not the installed one. You are confusing imports.